app/models/derived_variables/lettings_log_variables.rb

```diff
@@ -31,8 +31,8 @@ module DerivedVariables::LettingsLogVariables
   def scheme_has_multiple_locations?
     return false unless scheme
 
-    @scheme_locations_count ||= scheme.locations.active_in_2_weeks.size
-    @scheme_locations_count > 1
+    scheme_locations_count = scheme.locations.active_in_2_weeks.size
+    scheme_locations_count > 1
```

Comment:
Would it make sense to recalculate it only if `scheme_was != scheme` ?

Reply:
Potentially, although I was thinking that if someone added a location to the scheme that could also change this (that'll become more relevant in Manny's current work).

Reply:
Hmm, I think I understand, I'm curious whether this would cause much of a difference in terms of performance. Happy with this if that's the correct thing to do tho